google/cloud/bigtable/table.py

```diff
@@ -1130,11 +1143,21 @@ def _do_mutate_retryable_rows(self):
                 retry=None,
                 **kwargs
             )
-        except RETRYABLE_MUTATION_ERRORS:
+        except RETRYABLE_MUTATION_ERRORS as exc:
             # If an exception, considered retryable by `RETRYABLE_MUTATION_ERRORS`, is
             # returned from the initial call, consider
             # it to be retryable. Wrap as a Bigtable Retryable Error.
-            raise _BigtableRetryableError
+            # For InternalServerError, it is only retriable if the message is related to RST Stream messages
+            if (
+                isinstance(exc, InternalServerError)
+                and exc.message in RETRYABLE_MUTATION_ERRORS
```

Comment:
can we do exc.message.toLowerCase and update RETRYABLE_MUTATION_ERRORS to have {rst_stream, rst stream}? Because the error message might be slightly different, but "rst_stream" and "rst stream" should cover all cases.

Reply:
I added `.lower()` and `rst stream` to `RETRYABLE_INTERNAL_ERROR_MESSAGES`